packages/Modal/src/components/Content/Content.styles.js

```diff
@@ -0,0 +1,17 @@
+import styled from "styled-components";
+import stylers from "@paprika/stylers";
+import { spacer } from "@paprika/stylers/lib/helpers";
+
+const SPACE = spacer(2);
```

Comment:
this variable isnt needed

Reply:
Oh my change couldn't have been pushed..

Reply:
i think it is resolved, i needed to refresh

Reply:
Oh i already updated this https://github.com/acl-services/paprika/pull/921/files#diff-3371c2ca8904e71ec6ae31a706a9566fdcd72674217750415c25a93d261f1ea6R1